unity/Holo/Assets/Scripts/models_collection/VolumetricLoader.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Collections;
```

Comment:
Unity warns of inconsistent line endings that can lead to compile errors in the future.

Reply:
Done